http/http_client.hpp

```diff
@@ -364,7 +367,7 @@ class ConnectionInfo : public std::enable_shared_from_this<ConnectionInfo>
         }
 
         timer.cancel();
-        if (ec)
+        if (ec && ec != boost::asio::ssl::error::stream_truncated)
```

Comment:
There is a parser check missing in line 380

Reply:
This as is matches what we have upstream. 
https://github.com/openbmc/bmcweb/blob/1fccd0d5eafd56049a8a381543dd83d5559fbf25/http/http_client.hpp#L365

I don't understand why we want something different. @abhilashraju ?

Reply:
If we want something different, then let's do that upstream